internal/cli/plugin/plugin_asset.go

```diff
@@ -124,55 +118,37 @@ func (opts *AssetOpts) getPluginAssetAsReadCloser() (io.ReadCloser, error) {
 }
 
 func parseGithubReleaseValues(arg string) (*GithubRelease, error) {
-	parts := strings.Split(arg, "@")
-
-	owner, name, err := parseGithubRepoValues(parts[0])
+	regexPattern := `^(^(http|https)://(www\.)?(github\.com/)|^(www\.)?(github\.com/))?(?P<owner>[A-Za-z0-9.\-_]+)/(?P<name>[A-Za-z0-9.\-_]+)(/)?(@(?P<version>v?(\d+)(.\d+)?(.\d+)?|latest))?$`
```

Comment:
I think your regex could be improved a bit:
```
^((https?://(www\.)?)?github\.com/)?(?P<owner>[\w.\-]+)/(?P<name>[\w.\-]+)/?(@(?P<version>v?(\d+)(\.\d+)?(\.\d+)?|latest))?$
```

Changes:
- Made it a little shorter/readable
- Replaced `a-zA-Z0-9` with `\w`
- Fixed forgotten `\` in front of `.` in the version part

Reply:
yeah that makes sense, thanks!